examples/java/pom.xml

```diff
@@ -283,6 +286,33 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-flink_2.10</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+
+    <dependency>
```

Comment:
can you please separate this into 2 PRs? I think we need to handle and review Spark and Flink separately.

Per our discussion 2/3 weeks ago, Flink should be relatively easy and should be done first.